Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
```

Comment:
Could you please select a lighter base image?

Reply:
Should I use Alpine base image and then install python3 on it?

Reply:
never mind, found out alpine image for python3.8 and replaced it.